_maps/map_files/CorgStation/CorgStation.dmm

```diff
@@ -70674,7 +70674,7 @@
 "xhM" = (
 /obj/structure/marker_beacon,
 /turf/open/floor/plating{
-	icon_state = "panelscorched"
```

Comment:
fun fact: You can use `UpdatePaths` to replace these varedits! It'll help out mappers if you wanna throw one in

This should work:
```
/turf/open/floor/@SUBTYPES{icon_state = "panelscorched" : /turf/open/floor/@SUBTYPES{@OLD; burnt = 1; icon_state = @SKIP}
```

Reply:
:+1: